framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -986,7 +1039,8 @@ private void handleExpr(AnnotationBuilder builder, String name,
                     builder.setValue(name, arr);
                 }
             } else {
-                ErrorReporter.errorAbort("StubParser: unhandled annotation attribute type: " + expr + " and expected: " + expected);
+                ErrorReporter.errorAbort("StubParser: unhandled annotation attribute type: " + expr +
+                        " and expected: " + expected);
```

Comment:
I don't like reaching the end of the line and not knowing what happens next.
This is the style used most commonly in javac and the Checker Framework.